package.json

```diff
@@ -23,6 +23,7 @@
     "LICENSE",
     "README.md"
   ],
+  "type": "module",
```

Comment:
Original change suggested in https://github.com/probablyup/markdown-to-jsx/pull/498.